1.6.0-alpha.0 Build failed with errors

### Expected behavior
Build should work as expected without errors.
### Actual behavior
```
node src/index.js -c config.json
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module 'istanbul-middleware'
```
### Steps to reproduce
https://lisk-releases.ams3.digitaloceanspaces.com/lisk-core/lisk-1.6.0-alpha.0-5ea1bd4-Linux-x86_64.tar.gz
### Which version(s) does this affect? (Environment, OS, etc...)
1.6.0

--- a/framework/src/modules/http_api/init_steps/setup_servers.js
+++ b/framework/src/modules/http_api/init_steps/setup_servers.js
@@ -20,13 +20,13 @@ const express = require('express');
 const http = require('http');
 const https = require('https');
 const socketIO = require('socket.io');
-// eslint-disable-next-line import/no-extraneous-dependencies
-const im = require('istanbul-middleware');
 
 module.exports = ({ components: { logger }, config }) => {
 	const expressApp = express();
 
 	if (config.coverage) {
+		// eslint-disable-next-line import/no-extraneous-dependencies
+		const im = require('istanbul-middleware');
 		logger.debug(
 			'Hook loader for coverage - Do not use in production environment!'
 		);

